recipes/configs/llama2/7B_full_single_device.yaml

```diff
@@ -48,15 +48,15 @@ resume_from_checkpoint: False
 
 # Fine-tuning arguments
 batch_size: 2
-epochs: 3
+epochs: 1
 optimizer:
-  _component_: torch.optim.SGD
+  _component_: bitsandbytes.optim.PagedAdamW
   lr: 2e-5
```

Comment:
Why does mistral use 5e-6 but llama uses a different LR?

Reply:
Mistral FT hyperparams have not really been extensively tuned, cc @kartikayk who may have more context there